app/core/less/variables.less

```diff
@@ -26,6 +26,7 @@
 @purple-highcontrast:    #291A45;
 @purple-success:         #7950ea;
 @purple-blip:            #4E4490;
+@purple-uploader:        #291448;
```

Comment:
Was this where there was a question about what purple to use? I wonder if this is supposed to match [the darkest purple we use in the dataviz](https://github.com/tidepool-org/viz/blob/master/src/styles/colors.css#L28) (for avg on smbg trends and median on cbg trends). No action item for now, but just raising awareness among @hntrdglss @krystophv @jh-bate that we should pay attention to this and try to align blip & viz at the next opportunity if indeed there is a mismatch as I suspect.